frontend/src/components/widgets/ReviewersTable.jsx

```diff
@@ -2,24 +2,36 @@ import React from "react";
 import Table from "@material-ui/core/Table";
 import TableBody from "@material-ui/core/TableBody";
 import TableRow from "@material-ui/core/TableRow";
-import TableCell from "@material-ui/core/TableCell";
+import MuiTableCell from "@material-ui/core/TableCell";
 import Typography from "@material-ui/core/Typography";
 import Avatar from "@material-ui/core/Avatar";
 import Tooltip from "@material-ui/core/Tooltip";
 import Checkbox from "@material-ui/core/Checkbox";
-import { makeStyles } from "@material-ui/core/styles";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
 import UserAvatarLink from "./UserAvatarLink";
 
+const TableCell = withStyles({
+  root: {
+    borderBottom: "none",
+  },
+})(MuiTableCell);
+
 const useStyles = makeStyles((theme) => {
   return {
     reviewer: {
       width: theme.spacing(3),
       height: theme.spacing(3),
     },
+    table: {
+      height: "20vh",
```

Comment:
please take this away

Reply:
If you want to handle the layout then rather do it in the file where the layout happens. Meaning the card details page can decide how much space to give everything.  It'll be much more predicatable and it wont have consewuences in other components you aren't meaning to edit. 

Eg, have you thought about the other components that make use of this component? Eg the review queue? How is that effected? Did you mean to effect it? 

It's better to write code that on;ly effects what you intend to effect.

Reply:
taken out